flake: TestDelNode failed

Running `godep go test k8s.io/kubernetes/pkg/controller/node -v` show failures at times:

```console
$ godep go test k8s.io/kubernetes/pkg/controller/node -v
...
=== RUN   TestDelNode
--- FAIL: TestDelNode (0.00s)
	rate_limited_queue_test.go:76: Invalid queue. Got [0xc820251c00 0xc820251bc0], expected [second third]
...
```

The expected queue is `[second third]` while got `[third second]`, see code here: https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/node/rate_limited_queue_test.go#L76

I would give an idea that these cases have something similar, so referring #24784 and #24788 

/cc @gmarek

--- a/pkg/controller/node/rate_limited_queue_test.go
+++ b/pkg/controller/node/rate_limited_queue_test.go
@@ -62,6 +62,13 @@ func TestAddNode(t *testing.T) {
 }
 
 func TestDelNode(t *testing.T) {
+	defer func() { now = time.Now }()
+	var tick int64
+	now = func() time.Time {
+		t := time.Unix(tick, 0)
+		tick++
+		return t
+	}
 	evictor := NewRateLimitedTimedQueue(flowcontrol.NewFakeAlwaysRateLimiter())
 	evictor.Add("first")
 	evictor.Add("second")

